packages/ai-constructs/package.json

```
@@ -38,7 +38,7 @@
     "typescript": "^5.0.0"
   },
   "peerDependencies": {
-    "aws-cdk-lib": "^2.152.0",
+    "aws-cdk-lib": "2.152.0",
```

```suggestion
    "aws-cdk-lib": "^2.158.0",
```

By looking at package lock changes this is a new baseline for CDK.

Please do the following while bumping CDK:
1. Pick lowest minimum that works (seems `^2.158.0`)
2. Maintain `^` in every declaration.
3. Add changesets for all packages that were changed.